release-notes/swan-lake-beta4-release-note.md

```diff
@@ -273,6 +273,52 @@ function parse(string str) returns int? { // Now results in a warning.
 }
 ```
 
+##### Allow to use field access for optional fields in Records
+
+Now optional fields in Records can be accessed using Field Access expressions.
+
+- If a Record has an optional field and the type of the field does not include `nil`. It can be accessed using field access expression.
+
+```ballerina
+type A record {
+   int a;
+   int b?;
+   int? c?;
+};
+
+public function main() {
+   A a = {a:1, b:2, c:3};
+   int? b = a.b; // now this is allowed
+}
+```
```

Comment:
````suggestion
```

But the following is still not allowed since the field's type includes nil.

```ballerina
string? department = employee.department; // error
```
````